kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/k8s/HelmClient.java

```diff
@@ -152,4 +162,38 @@ private String mapToValuesParameterArgument(Map<String, String> m) {
     private static String concatEntry(Map.Entry<String, String> entry) {
         return String.format("%s=%s", entry.getKey(), entry.getValue());
     }
+
+    /**
+     * Gets chart version.
+     *
+     * @param chartName the chart name
+     * @param releaseVersion the release version
+     * @return the chart versions
+     */
+    public String getChartVersion(String chartName, String releaseVersion) {
+        String chartVersion = "";
+        if (releaseVersion != null && !releaseVersion.isEmpty()) {
+            LOGGER.info("Getting chart version for {}", chartName);
+            ExecResult result = Exec.exec(null, command("search", "repo",
+                    chartName,
+                    "--versions",
+                    "--output=json"), 0, true);
```

Comment:
if we know the chart version we are looking for we can use `String.format("--version ^%s", chartVersion)` which should do a semver aware search see the [helm docs](https://helm.sh/docs/helm/helm_search_repo/)